LPFilter/Source/PluginProcessor.cpp

```diff
@@ -167,15 +176,36 @@ void LpfilterAudioProcessor::processBlock (AudioSampleBuffer& ioBuffer, MidiBuff
     }
 }
 
-void LpfilterAudioProcessor::process(dsp::ProcessContextReplacing<float> context) noexcept
+void LpfilterAudioProcessor::process (AudioSampleBuffer& processBuffer) noexcept
 {
+    // Define the block that passes into process function
+    dsp::AudioBlock<float> block (processBuffer);
+    
     // lpfJuce filter processing
-    lpfJuce.process(context);
+   // lpfJuce.process(dsp::ProcessContextReplacing<float> (block));
+    
+    if (getTotalNumInputChannels() < 2 )
+    {
+        tempBuffer.copyFrom(0, *tempBuffer.getReadPointer(0), processBuffer.getReadPointer(0), processBuffer.getNumSamples());
```

Comment:
Instead of calling `getReadPointer ` you can simply write the correct sample.
For example, replace
```c++
tempBuffer.copyFrom(0, *tempBuffer.getReadPointer(0), processBuffer.getReadPointer(0), processBuffer.getNumSamples());
```
with 
```c++
tempBuffer.copyFrom(0, 0, processBuffer.getReadPointer(0), processBuffer.getNumSamples());
```